### DISCUSSION TITLE
What do I set for Github Endpoint?

### DISCUSSION THREAD

**Discussion by elalo-deloitte** [renovatebot/renovate#19404] [Request Help] [answered]
I've successfully run Renovate in my self-hosted Gitlab CI with the following command:

```
      podman run renovate/renovate:latest \
        --platform=gitlab \
        --token=$TOKEN \
        --endpoint=https://gitlab.dev.com/api/v4 \
        some_path/some_repo
```

However, when I try to run the same thing on my public Github, I get a 404.

```
      podman run renovate/renovate:latest \
        --platform=github \
        --token=$TOKEN \
        --endpoint=https://github.com/api/v3 \
        some_path/some_repo
```

I've also tried

```
      podman run renovate/renovate:latest \
        --platform=github \
        --token=$TOKEN \
        --endpoint=https://github.com.com/My_Company/api/v3 \
        some_path/some_repo
```

The one thing I'm wondering... If the PAT is attached to my user, should I include my username in the URL?

I would love some advice, thanks!

**Comment by viceice** [comment, marked as answer]
don't set platform and endpoint for GitHub, then working default's are used.

**Reply by elalo-deloitte** [reply]
Oh, wow. That definitely did it. Is that in the docs somewhere? 

One thing left... So I don't get an actual `404` anymore, but I now get `Repository is not found`.

```
 INFO: Repository started (repository=Company/Repo)
       "renovateVersion": "34.56.0"
ERROR: Repository is not found (repository=Company/Repo)
 INFO: Repository finished (repository=Company/Repo)
       "cloned": undefined,
       "durationMs": 343
 INFO: Renovate is exiting with a non-zero code due to the following logged errors
       "loggerErrors": [
         {
           "name": "renovate",
           "level": 50,
           "logContext": "_G4lMB-kStzUyQ-4ZwlaO",
           "repository": "Company/Repo",
           "msg": "Repository is not found"
         }
       ]
```

Any ideas?

**Reply by viceice** [reply]
set log level to debug, check the docs

**Reply by rarkins** [reply]
Something like `-e LOG_LEVEL=debug`

**Reply by elalo-deloitte** [reply]
Does it make a difference if the repository is private? 

In my original question I wrote "public Github" but I was trying to convey that it's not self-hosted. I'm realizing now that this could be confusing.

I've added the PAT to my user and I have access to the repo. So I'm not really sure what I'm missing. I've looked over the docs but I didn't see any mention of private repos making a difference.

Thanks,

**Reply by viceice** [reply]
it doesn't make a difference as lont as the PAT has access to that repo
